avm/utilities/pipelines/e2eValidation/resourceRemoval/Initialize-DeploymentRemoval.ps1

```diff
@@ -92,6 +92,7 @@ function Initialize-DeploymentRemoval {
             'Microsoft.Compute/virtualMachines',
             'Microsoft.VirtualMachineImages/imageTemplates', # Must be removed before their MSI
             'Microsoft.ManagedIdentity/userAssignedIdentities',
+            'Microsoft.Databricks/workspaces',
```

Comment:
Tried and failed hard

Reply:
Like the pipeline failed? Or the RGs remained? My 3 tests so far worked, but each time the Databricks deployment itself had issues

Reply:
The RG remained. Let's maybe see if the behavior changed and this workaround works now. If still not, as discussed, we can work on a custom post removal logic.

Reply:
Re-running one last time in the hopes that it's not getting stuck on the dependency issue again - like for the last 4 times.

Reply:
Can't get around the issue during the deployment. Will close the PR for now